src/undo.h

```diff
@@ -23,14 +26,22 @@ class CTxInUndo
     bool fCoinBase;       // if the outpoint was the last unspent: whether it belonged to a coinbase
     unsigned int nHeight; // if the outpoint was the last unspent: its height
     int nVersion;         // if the outpoint was the last unspent: its version
+    uint256 originScId;   // if the outpoint was the last unspent: its originScId, introduced with certificates
```

Comment:
add a backwardTrasferStartIdx field that will be serialized only in case of Coins coming from certificates that had been just pruned.